src/main/resources/db.changelog.app_user-1.0.0.xml

```diff
@@ -1,51 +1,51 @@
 <databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd">
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.22.xsd">
 
-    <changeSet id="ticket-1.0.0" author="Janus Chung" failOnError="true">
-        <comment>create table to store user information </comment>
-        <createTable tableName="USER" schemaName="support_system">
+    <changeSet id="app_user-1.0.0" author="Janus Chung" failOnError="true">
+        <comment>create table to store app user information </comment>
+        <createTable tableName="APP_USER">
             <column autoIncrement="true" name="ID" type="BIGINT" remarks="Primary key">
-                <constraints primaryKey="true"/>
+                <constraints primaryKey="true" nullable="false"/>
```

Comment:
Doesn't hurt anything, but a Primary Key by definition is a Unique Constraint plus a NOT NULL constraint, so no need to set nullable="false"